vnext/include/ReactUWP/XamlView.h

```diff
@@ -30,4 +31,22 @@ inline void SetTag(XamlView view, winrt::IInspectable tag)
   SetTag(view, tag.as<winrt::IPropertyValue>().GetInt64());
 }
 
+inline bool IsValidTag(winrt::IPropertyValue value)
+{
+  assert(value);
```

Comment:
How useful are these asserts? Isn't the next line going to obviously AV?

Reply:
_In_ doesn't work in this situation. assert just make the user knows that we should not pass nullptr. I will keep assert here instead of null crash

---
In reply to: [295560591](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295560591) [](ancestors = 295560591)